usb.go

```diff
@@ -210,6 +230,38 @@ func (c *Context) OpenDevices(opener func(desc *DeviceDesc) bool) ([]*Device, er
 	return ret, reterr
 }
 
+// OpenDeviceWithFileDescriptor takes a (Unix) file descriptor of an opened USB device
+// and wraps the library around it.
+// This is particularly useful when working on Android, where the USB device must be opened
+// by the SDK (Java), giving access to the device through the file descriptor (https://developer.android.com/reference/android/hardware/usb/UsbDeviceConnection#getFileDescriptor()).
+//
+// Do note that for this to work the automatic device discovery must be disabled, by calling
```

Comment:
not "by calling", but perhaps:

discovery must be disabled at the time when the new Context is created, through the use of ContextOptions.DeviceDiscovery. Example: